### DISCUSSION TITLE
Fade-out - blank - Fade-in

### DISCUSSION THREAD

**Discussion by ghost** [helgeerbe/picframe#118] [General]
Hi.  I've been experimenting with the config settings to see if I can setup behavior where there is no cross over between images and in fact a blank of a few seconds.  The reason is that I think this pause would be nice when displaying paintings i.e. no 'contamination' of another painting, for want of a better word.  I think the cross over is great for photo's.
Well, I tried setting fade-time to 0, even negative numbers with interesting results (0 stopped the app from running I found when resetting the service).
Is there an easy way to do this?
I love this app BTW, thanks for all your efforts!

Best Regards

**Comment by paddywwoof** [comment]
@TraffordSt There isn't anything built in as it stands. I will think a bit more about what might be the easiest way to achieve this, but it will involve changing the code of https://github.com/helgeerbe/picframe/blob/main/picframe/viewer_display.py a bit

**Reply by ghost** [reply]
Thanks @paddywwoof
